mxnet/training/docker/1.8/py3/Dockerfile.cpu

```diff
@@ -106,15 +106,15 @@ WORKDIR /
 
 RUN ${PIP} install --no-cache --upgrade \
```

Comment:
This file has no changes. It is like the current master. The change has been made to avoid conflict.